proxy/http2/Http2ClientSession.cc

```diff
@@ -326,6 +327,9 @@ Http2ClientSession::main_event_handler(int event, void *edata)
     break;
 
   case VC_EVENT_WRITE_COMPLETE:
+    if (this->get_half_close_flag()) {
```

Comment:
@sidhuagarwal 

1. Yes, we should stop accepting new streams. I think we can respond with RST_STREAM frame for HEADERS frames newly arrived.
2. Yes and no. Not only. Basically we should keep processing streams already open as long as possible, as you commented. However, if one of streams were transferring gigabytes file, it may take long time. In this case we want to close the stream even if it's not completed. This is the reason I mentioned active timeout.